src/herder/HerderImpl.cpp

```diff
@@ -762,6 +765,25 @@ HerderImpl::triggerNextLedger(uint32_t ledgerSeqToTrigger)
         }
     }
 
+    if (!upgrades.empty())
+    {
+        auto message =
+            fmt::format("Proposing herder configuration upgrades: {0}; network "
+                        "values for herder are: ledgerVersion={1}, "
+                        "baseFee={2}, baseReserve={3}, maxTxSetSize={4}",
+                        Upgrades::toString(upgrades), lcl.header.ledgerVersion,
+                        lcl.header.baseFee, lcl.header.baseReserve,
+                        lcl.header.maxTxSetSize);
+        CLOG(WARNING, "Herder") << message;
```

Comment:
I think it should be `INFO` (INFO > WARNING), we want this to be nagging/highly visible